index.html

```diff
@@ -0,0 +1,857 @@
+                atque.
+              </p>
+            </div>
+            <div class="app-small-img">
+              <img src="./images/l-2.jpg" alt="app 1 image" />
+            </div>
+          </div>
+          <div class="app-features mb-1">
+            <ul>
+              <li class="pb-0-8">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro,
+                laboriosam.
+              </li>
+              <li class="pb-0-8">
+                Lorem ipsum dolor sit amet consectetur adipisicing.
+              </li>
+              <li class="pb-0-8">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              </li>
+              <li class="pb-0-8">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Nihil?
+              </li>
+              <li class="pb-0-8">Lorem ipsum dolor sit amet.</li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="app">
+          <h2 class="heading2 orange">Distract ></h2>
+          <div class="header-img hero-img-1 mb-0 tab-img">
+            <div class="overlay tab-overlay">
+              <h4>Most popular</h4>
+              <p class="app-img-text tab-img-text">
+                Netflix will cancel account if you havent used itin a while
+              </p>
+              <span>Author Updated 10 hours ago</span>
+            </div>
+          </div>
+          <div class="below-app-image mb-1">
+            <div class="description">
+              <h4 class="orange">LATEST</h4>
+              <p class="orange">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Fuga,
+                atque.
+              </p>
+            </div>
+            <div class="app-small-img">
+              <img src="./images/l-4.jpg" alt="app 1 image" />
+            </div>
+          </div>
+          <div class="app-features mb-1">
+            <ul>
+              <li class="pb-0-8">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro,
+                laboriosam.
+              </li>
+              <li class="pb-0-8">
+                Lorem ipsum dolor sit amet consectetur adipisicing.
+              </li>
+              <li class="pb-0-8">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit.
+              </li>
+              <li class="pb-0-8">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Nihil?
+              </li>
+              <li class="pb-0-8">Lorem ipsum dolor sit amet.</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        LATEST DEALS SECTION
+      +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+
+      <section class="latest-deals">
+        <div class="latest-deals-header">
+          <h2 class="heading2 orange inline-block">Latest Deals</h2>
```

Comment:
- [ ] Kindly check the images size for medium screens.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83334201-6b616700-a26a-11ea-9284-a4f18c2820d3.png) | ![image](https://user-images.githubusercontent.com/57414789/83334184-58e72d80-a26a-11ea-87a7-2a8a8f362cda.png)

Reply:
All changes have been fixed as suggested